patches/man.c.patch

```diff
@@ -29,3 +29,31 @@ index 761f4b4..4b861be 100644
  		pipecmd *iconv_cmd;
  		iconv_cmd = pipecmd_new_args
  			("iconv", "-c", "-f", source, "-t", target_translit,
+@@ -4417,8 +4423,25 @@ int main (int argc, char *argv[])
+ 					if (!section && maybe_section &&
+ 					    CTYPE (isdigit, nextarg[0]))
+ 						gripe_no_name (nextarg);
+-					else
+-						gripe_no_man (nextarg, section);
++					else {
++#ifdef __MVS__
++            char* fmt = "/bin/man %s %s 2>/dev/null";
```

Comment:
Nice, curious why you are suppressing the stderr output?

Reply:
I should probably document that in a comment. If you say ‘man jack’ you end up with two errors that it can’t find it if I don’t suppress stderr

Reply:
Ah!